x/crisis/keeper.go

```diff
@@ -17,15 +17,15 @@ type Keeper struct {
 	paramSpace     params.Subspace
 	invCheckPeriod uint
 
-	distrKeeper         DistrKeeper
-	bankKeeper          BankKeeper
-	feeCollectionKeeper FeeCollectionKeeper
+	distrKeeper         distrKeeper
+	bankKeeper          bankKeeper
+	feeCollectionKeeper feeCollectionKeeper
 }
 
 // NewKeeper creates a new Keeper object
 func NewKeeper(paramSpace params.Subspace, invCheckPeriod uint,
```

Comment:
Don't we want the keeper to also be internal? Currently, this is inconsistent with the changes made on `bank`.

Reply:
I'll move it across `internal`